src/views/2_SessionComponent.tsx

```diff
@@ -25,6 +25,10 @@ export class SessionComponent extends React.Component<Props, {}> {
     }
 
     componentDidMount() {
+        this.session.onscroll = () => {
+            const shutterOffset = this.session.scrollTop;
+            this.shutter.style.transform = `translateY(${shutterOffset}px)`;
+        };
```

Comment:
I remembered that it was implemented already, but because of lack of tests I broke it later somehow. Please refer to this discussion and also take a look at where `jobsContainer` is used in that PR. https://github.com/shockone/black-screen/pull/580/files#r70489211